src/focus.service.ts

```diff
@@ -508,7 +508,7 @@ export class FocusService {
    */
   private isFocusable(el: Element) {
     const role = el.getAttribute('role');
-    const tabIndex = el.getAttribute('tabIndex');
+    const tabIndex = el.getAttribute('tabindex');
```

Comment:
Are you sure? `el` comes from `querySelectorAll` and the interface for that is `querySelectorAll(selectors: string): NodeListOf<Element>` (which we can't change). There might be some good reason why it's returning an Element and not a HTMLElement?